src/main/java/appeng/me/cache/TickManagerCache.java

```diff
@@ -66,7 +60,7 @@ public long getAvgNanoTime(final IGridNode node) {
             return -1;
         }
 
-        return tt.getAvgNanos();
```

Comment:
Since this method always return 0, it kinda draws the entire point of this method into question...

Reply:
I think this was never finished. I might have improved it when considering adding time dilation to grid ticking, but that was never merged. One of the debug tool makes use of it, but currently it is useless.